rekall/plugins/addrspaces/amd64.py

```diff
@@ -105,6 +105,39 @@ def get_one_gig_paddr(self, vaddr, pdpte):
 
     lock = 0
 
+    def vaddr_access(self, vaddr):
+        """Is the access bit set on the page for the vaddr?"""
```

Comment:
Oh I see - you are and'ing all the values together. Maybe return False instead of None then.